src/webloader.js

```diff
@@ -142,6 +143,8 @@ function main() {
 
 	// If uid is empty, or evolvLazyUid is set - don't run the webloader until a uid is set using evolv.setUid().
```

Comment:
The gaIntegration should only execute if the flag is set. Not if it UID is not set.

Reply:
fixed